tests/engine/test_process_function.py

```diff
@@ -352,7 +352,7 @@ def test_launchers(self):
         self.assertTrue(isinstance(node, orm.CalcFunctionNode))
 
         # Process function can be submitted and will be run by a daemon worker as long as the function is importable
-        # Note that the actual running is not tested here but is done so in `.ci/test_daemon.py`.
+        # Note that the actual running is not tested here but is done so in `.github/ci_only_tests/test_daemon.py`.
```

Comment:
```suggestion
        # Note that the actual running is not tested here but is done so in `.github/system_tests/test_daemon.py`.
```

Reply:
maybe do a search/replace on `ci_only` => `system`

Reply:
yep just done and no ither places